setup/ubuntu/bootstrap.sh

```diff
@@ -157,6 +157,10 @@ apt-get install -y libffi-dev libssl-dev
 # MySQL dependencies:
 apt-get install -y libmysqlclient-dev
 
+# Microsoft SQL Server dependencies (`sudo` required):
+sudo apt-get install -y freetds-dev
```

Comment:
You should add this to the Dockerfile as well, although I can do that myself if you prefer.